app/main/routes.py

```diff
@@ -102,18 +102,34 @@ def browse():
     consignment_id = request.args.get("consignment_id", None)
 
     browse_type = "browse"
+    filters = {}
+    date_from = None
+    date_to = None
 
     if transferring_body_id:
         browse_type = "transferring_body"
-        filters = {"transferring_body_id": transferring_body_id}
+        filters["transferring_body_id"] = transferring_body_id
     elif series_id:
         browse_type = "series"
-        filters = {"series_id": series_id}
+        filters["series_id"] = series_id
     elif consignment_id:
         browse_type = "consignment"
-        filters = {"consignment_id": consignment_id}
-    else:
-        filters = {}
+        filters["consignment_id"] = consignment_id
+        # date_from = request.args.get("date_from", '01/02/2023')
+        # date_to = request.args.get("date_to", '27/02/2023')
+    # else:
+    # for browse
+    # date_from = request.args.get("date_from", '01/10/2023')  #
+    # date_to = request.args.get("date_to", '31/10/2023')  #
+
+    date_range = {}
```

Comment:
as discussed offline, will remove this from this PR and leave it for fully integrating into the view, reading values from the html in another PR

Reply:
done the changes and removed the code from browse function in routes.py